cmd/config.go

```diff
@@ -504,20 +512,6 @@ func importCloudstackINI(option, csPath, cfgPath string) error {
 	return addAccount(cfgPath, config)
 }
 
-func doesAccountExist(name string) bool {
```

Comment:
The `doesAccountExist()` function has been deleted and replaced by the `getAccountByName()` function with which it shared most of its code.